csharp/ql/campaigns/Solorigate/src/qlpack.yml

```diff
@@ -0,0 +1,9 @@
+name: codeql/solorigate-queries
```

Comment:
should this mention `csharp`?

Reply:
Good point. I made the change. Unpublished previous packs and re-published under the new name.

Reply:
Did you forget to push the change?

Reply:
I pushed to the wrong remote. Should be good now.